packages/editor-ui/index.html

```diff
@@ -9,6 +9,7 @@
 			window.BASE_PATH = '/{{BASE_PATH}}/';
 			window.REST_ENDPOINT = '{{REST_ENDPOINT}}';
 		</script>
+		<script src="/{{REST_ENDPOINT}}/sentry.js"></script>
```

Comment:
So request the Sentry config even when Sentry is disabled?

Reply:
yeah. frontend build doesn't know about if sentry is enabled or disabled. When we release, the frontend code is compiled into a bunch of static files, that don't change according to any runtime configuration (except N8N_PATH).
So, this script needs to be injected here to be able to get this dynamic information into the frontend before anything else loads.